### PR TITLE
adds support for p5.Color in style()

### PR DESCRIPTION
**Pull request by futuremarc**
```
var col = color(25,23,200,50);
select('body').style('background-color', col);
```

### CODE REVIEW

File: lib/addons/p5.dom.js [hunk 1]
```diff
@@ -1136,13 +1136,17 @@
...
+   * var col = color(25,23,200,50);
    * myDiv.style("font-size", "18px");
+   * myDiv.style("background-color", col);
    * </code></div>
    */
   p5.Element.prototype.style = function(prop, val) {
     var self = this;
 
+    if (val instanceof p5.Color)
+      val = 'rgba(' + val.rgba[0] + ',' + val.rgba[1] + ',' + val.rgba[2] + ',' + val.rgba[3]/100 + ')'
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: I think this should be /255?

File: lib/addons/p5.dom.js [hunk 2]
```diff
@@ -1136,13 +1136,17 @@
    * @example
    * <div><code class="norender">
    * var myDiv = createDiv("I like pandas.");
-   * myDiv.style("color", "#ff0000");
+   * var col = color(25,23,200,50);
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: can we keep both examples here? ie using a color object or using a string?

**Comment by lmccart** [comment]
rad! just a couple small notes..

**Comment by futuremarc** [comment]
there you go!

**Comment by lmccart** [comment]
awesome thanks, so fast!

**Pull request merged**